<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartGame</class>
 <widget class="QMainWindow" name="StartGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1202</width>
    <height>793</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="button1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>260</y>
      <width>300</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Answer 1.</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="button2">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>260</y>
      <width>300</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Answer 2.</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="button3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>460</y>
      <width>300</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Answer 3.</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="button4">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>460</y>
      <width>300</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Answer 4.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>150</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
font: 20pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Question:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1202</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
